### ISSUE TITLE
A major update !

### ISSUE THREAD

**Issue by finscn** [finscn/Ejecta#27]
first , my fork version supports Xcode 7 & iOS 9 now.

second, 
I update some 3rd libs:
- SocketRocket ( it's same to https://github.com/square/SocketRocket )
- GoogleMobileAds  (  the lastest version )
- Chartboost (  the lastest version )

last, 
I tidy  launchImage and Icon , move all of them to Asset Set and remove unused file.

---

Now I don't have  iOS 7 and 8  device , So I can't test the compatibility ( for iOS 7 & 8 ) of  new version .

But I hope my fork version could run on   iOS  7 , 8 , 9 device .

So I need your help , thank you very much.

---

NOTICE 

Google AdMob doesn't supports  Bitcode (a new feature of iOS 9) , so I have to disable this feature.
After google AdMob supported , I will turn it on .

**Comment by finscn** [comment]
@matrixreal  , could you help me ? thanks

**Comment by matrixreal** [comment]
@finscn 
yes i will try it in couple of minutes

**Comment by matrixreal** [comment]
@finscn 
every thing seem to work good except one warning like picture below
![warning](https://cloud.githubusercontent.com/assets/6823974/9931692/3b244e8e-5d3e-11e5-9d5c-f2d6e5f4ac31.png)

**Comment by finscn** [comment]
@matrixreal ,  I updated . You could  try  again , thanks

**Comment by matrixreal** [comment]
@finscn 
i confirm everything is okay on iphone 4s ios 7.1.2

**Comment by matrixreal** [comment]
and what about ios9 and xcode 7 ? it run good ?

**Comment by finscn** [comment]
If you want to run your game on iOS 7 , I think Xcode 6.x is a better choice.

**Comment by matrixreal** [comment]
yes but i have update my games to ios9  i hope that it run good on ios9 and apple will not reject

**Comment by matrixreal** [comment]
@finscn 
you say that you update the socketrocket
what are the different call and condition of this extention please ?
thanks

**Comment by finscn** [comment]
No different . Use it as same as in original Ejecta

**Comment by matrixreal** [comment]
an example please ? the different call and condition .... as i'm plan to create the first multiplayer online game with your fork :dancer:

**Comment by finscn** [comment]
You could learn How to use websocket in  HTML5/Web . They are same.
